io.github.pemsley.coot.yaml

```diff
@@ -0,0 +1,511 @@
+        url: https://download.gnome.org/sources/gobject-introspection/1.82/gobject-introspection-1.82.0.tar.xz
+        sha256: 0f5a4c1908424bf26bc41e9361168c363685080fbdb87a196c891c8401ca2f09
+    post-install:
+      - ln -sf /app/libexec/bin/* /app/bin/
+
+  - name: boost-python3
+    sources:
+      - type: archive
+        url: https://github.com/boostorg/boost/releases/download/boost-1.83.0/boost-1.83.0.tar.gz
+        sha256: 0c6049764e80aa32754acd7d4f179fd5551d8172a83b71532ae093e7384e98da
+    buildsystem: simple
+    build-commands:
+      - echo "using python" > user-config.jam
+      - echo ": 3.12" >> user-config.jam
+      - echo ": /usr/bin/python3" >> user-config.jam
+      - echo ": /usr/include/python3.12" >> user-config.jam
+      - echo ": /usr/lib" >> user-config.jam
+      - echo ";" >> user-config.jam
+      - ./bootstrap.sh --prefix=/app --libdir=/app/lib --with-python=python3.12 --with-python-root=/usr --with-libraries=python
+      - ./b2 --build-dir=build-python3 --stagedir=stage-python3 --prefix=/app --libdir=/app/lib abi=sysv linkflags=-L/usr/lib --user-config=user-config.jam python=3.12 -d2 -j$(nproc) --layout=system threading=multi link=shared,static install
+    build-options:
+      env:
+        CXXFLAGS: "-std=c++14"
+        PYTHON: "/usr/bin/python3"
+        PYTHONINCLUDE: "/usr/include/python3.12"
+        PYTHONLIB: "/usr/lib"
+    cleanup:
+      - /app/share/boost
+
+  - name: libdwarf
+    sources:
+      - type: archive
+        url: https://www.prevanders.net/libdwarf-0.11.0.tar.xz
+        sha256: 846071fb220ac1952f9f15ebbac6c7831ef50d0369b772c07a8a8139a42e07d2
+    buildsystem: simple
+    build-options:
+      environment:
+        PKG_CONFIG_PATH: /app/lib/pkgconfig
+    build-commands:
+      - if [ "${FLATPAK_BUILDER}" = "true" ]; then sh autogen.sh; fi
+      - ./configure --prefix=/app --enable-shared
+      - make
+      - make install
+
+  - name: gsl
+    sources:
+      - type: archive
+        url: https://ftp.gnu.org/gnu/gsl/gsl-2.8.tar.gz
+        sha256: 6a99eeed15632c6354895b1dd542ed5a855c0f15d9ad1326c6fe2b2c9e423190
+    buildsystem: simple
+    builddir: true
+    build-commands:
+      - ./configure --prefix=/app
+      - make
+      - make install
+
+  - name: glm
+    sources:
+      - type: archive
+        url: https://github.com/g-truc/glm/archive/refs/tags/1.0.1.tar.gz
+        sha256: 9f3174561fd26904b23f0db5e560971cbf9b3cbda0b280f04d5c379d03bf234c
+    buildsystem: cmake
+    build-options:
+      build-tests: false
+    build-commands:
+      - cmake -S . -B build -DGLM_BUILD_TESTS=OFF -DBUILD_SHARED_LIBS=ON -DCMAKE_INSTALL_PREFIX=/app
+      - cmake --build build
+      - cmake --install build
+      - |
+        echo 'prefix=/app
+        includedir=${prefix}/include
+        libdir=${prefix}/lib
+
+        Name: GLM
+        Description: OpenGL Mathematics
+        Version: 1.0.1
+        Cflags: -I${includedir}' > /app/lib/pkgconfig/glm.pc
+
+  - name: gtk4
```

Comment:
why do you need to build gtk4?

Reply:
This app needs gtk4. Does Freedesktop SDK has gtk4?

Reply:
no

Reply:
use the GNOME runtime.

Reply:
I think GNOME runtime has extra packages not needed for this app.

Sould I use GNOME runtime?

Reply:
> use the GNOME runtime.

I don't think this was ambiguous.

Reply:
Yes, the GNOME runtime is the appropriate dependency - that much I know.